### ISSUE TITLE
Consider to use aria-selected on the highlighed option

### ISSUE THREAD

**Issue by afercia** [moroshko/react-autowhatever#31]
Ideally, the highlighted option in the list of suggestions should have an `aria-selected` state set to `true`. Quoting from the [Listbox section](https://www.w3.org/TR/wai-aria-practices/#Listbox) of the ARIA Authoring Practices:

> In a single-select listbox, the selected option has aria-selected set to true.

Apart from semantics, there's also a concrete side-effect: to my knowledge, Safari 10 + VoiceOver don't announce the highlighted option if `aria-selected` is not present.

**Comment by moroshko** [comment]
Thanks for raising this issue!
Mind creating a PR?
